doc/glossary.rst

```diff
@@ -0,0 +1,32 @@
+Glossary
+========
+
+.. currentmodule:: hnn_core
+
+The Glossary provides short definitions of HNN-specific vocabulary and
+general computational neuroscience concepts. If you think a term is missing,
+please consider creating a new issue or opening a pull request to add it.
+
+.. glossary::
+    :sorted:
+
+
+    drive
+        An artificial cell or population of artificial cells that only produces
+        spikes according to a given pattern and is used to initiate and sustain
+        cell spiking in the network
```

Comment:
Is "drive" an artificial cell or the synaptic spikes that emerge from an artificial cell? This is nuanced, but I think of "drive" as modeling a physiological process rather than a physiological structure.

Reply:
agreed, and we may even want to avoid saying "artificial cells" ... it's bringing another new term to explain it. Can you make a suggested edit?

Reply:
```suggestion
    drive
        An exogenous source of excitatory action potentials (spikes)
        that synapse onto neuronal compartments, perturbing the network.
```

Reply:
okay I deleted the second sentence to keep it simple for now. Thanks for the suggestion!